lib/node/nodes/contour.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var Node = require('../node');
+
+var TYPE = 'contour';
+
+var METHODS = ['nearest_neighbor', 'barymetric','IDW'];
+
+var CLASS_METHODS = ['equals', 'headstails', 'jenks', 'quantiles'];
+
+var PARAMS = {
+    // the scattered points dataset with real values
+    source: Node.PARAM.NODE(Node.GEOMETRY.POINT),
+    // column of values in source dataset
+    column: Node.PARAM.STRING(),
+    // buffer around my dataset
+    buffer: Node.PARAM.NULLABLE(Node.PARAM.NUMBER(), 0.2),
+    // 0: nearest neighbor, 1: barymetric, 2: IDW
+    method: Node.PARAM.NULLABLE(Node.PARAM.ENUM.apply(null,METHODS), METHODS[1]),
+    // classification method
+    class_method: Node.PARAM.NULLABLE(Node.PARAM.ENUM.apply(null,CLASS_METHODS), CLASS_METHODS[3]),
```

Comment:
Hmmm... but that way I'd need to declare several variables. One for each string... This way I only need to maintain two arrays that are clearly defined.

But, if you prefer one var per string, I'll change it.